src/Calendar.js

```diff
@@ -112,6 +112,24 @@ class Calendar extends React.Component {
     onSelectSlot: PropTypes.func,
 
     /**
+     * Callback fired when empty calendar cell was clicked twice.
+     *
+     * ```js
+     * (date: Date, e: SyntheticEvent) => any
+     * ```
+     */
+    onDoubleClickSlot: PropTypes.func,
+
+    /**
+     * Callback fired when empty calendar all-day cell was clicked twice.
+     *
+     * ```js
+     * (date: Date, e: SyntheticEvent) => any
+     * ```
+     */
+    onDoubleClickAllDaySlot: PropTypes.func,
```

Comment:
I don't think we have a normal `click` equivalent to this one?

Reply:
I decided to separate `onDoubleClickSlot` and `onDoubleClickAllDaySlot`. Because both of them can get same `date` parameter, but handling behavior is usually different for clicks on regular cells and all-day cells.
What do you think?